layouts/partials/head/twitter.html

```diff
@@ -0,0 +1,15 @@
+{{ if .Site.Params.Settings.pageTwitter}}
+<meta name="twitter:site" content="{{ .Site.Params.Settings.pageTwitter}}">
+{{ else }}
+<meta name="twitter:site" content="{{ .Site.Params.social.twitter}}">
+{{ end }}
+<meta name="twitter:creator" content="{{ .Site.Params.social.twitter}}">
+{{ if .IsPage }}
+<meta name="twitter:description" content="{{ .Summary }}" />
+<meta name="twitter:title" content="{{ .Title }}" />
+<meta name="twitter:card" content="summary_large_image" />
```

Comment:
For `twitter:card` tag, we can use this logic:

```html
<meta name='twitter:card' content='summary{{- if .Params.cover -}}_large_image{{- end -}}'>
```

This way, it'll be one less thing for the users to worry about. What do you think about it?

Reply:
Check this: http://minimo.netlify.com/docs/cover-image/

Reply:
Ah ok. Thought it was an inbuilt variable.